pylint/checkers/imports.py

```diff
@@ -712,6 +717,20 @@ def _check_deprecated_module(self, node, mod_path):
             if mod_path == mod_name or mod_path.startswith(mod_name + '.'):
                 self.add_message('deprecated-module', node=node, args=mod_path)
 
+    def _check_import_as_rename(self, node):
+        names = node.names
+        if not all(names[0]):
+            return
+
+        real_name = names[0][0]
```

Comment:
Sorry I didn't mention this before. We need to check the import alias for each import:

```python
>>> astroid.extract_node('from . import foo as bar, foo2 as bar2').names
[('foo', 'bar'), ('foo2', 'bar2')]
```

So we'll need to loop over everything in `names` and check each tuple.

Reply:
I will try these test cases.